packages/@pollyjs/core/package.json

```diff
@@ -52,8 +52,8 @@
     "@pollyjs/utils": "^0.1.1",
     "@sindresorhus/fnv1a": "^1.0.0",
     "blueimp-md5": "^2.10.0",
+    "fast-json-stable-stringify": "^2.0.0",
```

Comment:
This is used by a lot of our dependencies. It has no dependents and is proven to be faster.